inc/saitypes.h

```diff
@@ -463,6 +465,9 @@ typedef struct _sai_tunnel_map_params_t
     /** VNI id */
     sai_uint32_t vni_id;
 
+    /** Bridge IF */
+    sai_object_id_t bridge_if;
```

Comment:
it will be really hard to track this in metadata, it would be better if this would be an attribute